### PR TITLE
style: add breadcrumb clear

### PR DESCRIPTION
**Pull request by xrkffgg**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
none

### 💡 Background and solution
- 打开链接 https://ant.design/components/page-header-cn/#components-page-header-demo-breadcrumb
- 选择 RTL 模式
- 可以看到面包屑的位置塌陷
- 因为 RTL 模式下，面包屑使用了 float，这里加上了清除浮动

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Added Breadcrumb RTL mode to clear floats          |
| 🇨🇳 Chinese | 增加  Breadcrumb RTL 模式下清除浮动           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/style/mixins/reset.less [hunk 1]
```diff
@@ -10,4 +10,10 @@
   line-height: @line-height-base;
   list-style: none;
   font-feature-settings: @font-feature-settings-base;
+
+  &::after {
+    display: block;
+    clear: both;
+    content: '';
+  }
```
Comment (afc163) [inline components/style/mixins/reset.less]: 这个覆盖太广了，单独修改 breadcrumb 吧。

Comment (yoyo837) [reply components/style/mixins/reset.less]: 我刚想说 😄

File: components/breadcrumb/style/index.less [hunk 2]
```diff
@@ -9,6 +9,12 @@
   color: @breadcrumb-base-color;
   font-size: @breadcrumb-font-size;
 
+  &::after {
+    display: block;
+    clear: both;
+    content: '';
+  }
```
Comment (afc163) [inline components/breadcrumb/style/index.less]: 有个 .clearfix 的 mixin 可以用。

File: components/breadcrumb/style/index.less [hunk 3]
```diff
@@ -5,6 +5,7 @@
 
 .@{breadcrumb-prefix-cls} {
   .reset-component;
+  .clearfix;
```
Comment (afc163) [inline components/breadcrumb/style/index.less]: 给 rtl 模式单独加就可以了。

**Pull request merged**